Codeinwp/neve
# fix: createInterpolateElement for WP<5.5

### Summary
Added a check for `createInterpolateElement` and if not present I revert the External links to a form that does not use React component interpolation.
This only applies to WordPress < 5.5 if equal or higher it should remain the same.

### Will affect visual aspect of the product
NO


### Test instructions
1. Install WP v5.3.8
2. Install Neve theme
3. Go options page on WP admin -> appearance -> neve options
4. Check that it displays correctly.

Closes #2922.
